ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,169 @@
+      case disclosure
+      case button
+  }
+
+  public class MSFListTokens: ObservableObject {
+      @Published public var backgroundColor: UIColor!
+      @Published public var borderColor: UIColor!
+      @Published public var leadingIconColor: UIColor!
+      @Published public var trailingIconColor: UIColor!
+      @Published public var leadingTextColor: UIColor!
+      @Published public var trailingTextColor: UIColor!
+
+      @Published public var borderSize: CGFloat!
+      @Published public var padding: CGFloat!
+      @Published public var textFont: UIFont!
+      @Published public var subtitleFont: UIFont!
+      @Published public var leadingIconSize: CGFloat!
+      @Published public var trailingIconSize: CGFloat!
+      @Published public var interspace: CGFloat!
+
+      var accessoryStyle: MSFListAccessoryVnextStyle!
+      var accessorySize: MSFListAccessoryVnextSize!
+
+      public init(accessoryStyle: MSFListAccessoryVnextStyle,
+                  accessorySize: MSFListAccessoryVnextSize) {
+          self.accessoryStyle = accessoryStyle
+          self.accessorySize = accessorySize
+          self.themeAware = true
+
+          didChangeAppearanceProxy()
+      }
+
+      @objc open func didChangeAppearanceProxy() {
+          backgroundColor = appearanceProxy.backgroundColor
+          borderColor = appearanceProxy.borderColor
+          borderSize = appearanceProxy.borderSize
+          padding = appearanceProxy.padding
+          textFont = appearanceProxy.textFont
+          subtitleFont = appearanceProxy.subtitleFont
+          leadingTextColor = appearanceProxy.textColor.leading
+          trailingTextColor = appearanceProxy.textColor.trailing
+
+          switch accessoryStyle {
+          case .iconOnly, .none:
+              leadingIconColor = appearanceProxy.iconColor.iconOnly
+              trailingIconColor = appearanceProxy.iconColor.iconOnly
+          case .withLabel:
+              leadingIconColor = appearanceProxy.iconColor.withLabel
+              trailingIconColor = appearanceProxy.iconColor.withLabel
+          case .disclosure:
+              trailingIconColor = appearanceProxy.iconColor.disclosure
+          }
+
+          switch accessorySize {
+          case .icon, .none:
+              leadingIconSize = appearanceProxy.iconSize.icon
+              trailingIconSize = appearanceProxy.iconSize.icon
+              interspace = appearanceProxy.interspace.icon
+          case .persona:
+              leadingIconSize = appearanceProxy.iconSize.persona
+          case .disclosure:
+              trailingIconSize = appearanceProxy.iconSize.disclosure
+              interspace = appearanceProxy.interspace.disclosure
+          case .button:
+              interspace = appearanceProxy.interspace.button
+          }
+      }
+  }
+
+  public struct MSFListView: View {
+      @ObservedObject var state: MSFListItem
+      @ObservedObject var tokens: MSFListTokens
+      var numCells: [MSFListItem]
```

Comment:
why numCells? isn't this the entire collection of cells and not only the number of cells?